manager/base.py

```diff
@@ -31,6 +31,7 @@
 # Prometheus support
 # Counter for all-the-get-calls, dealt with in BaseHandler
 REQUEST_COUNTS = Counter('ve_manager_invocations', 'Number of calls per handler', ['method', 'endpoint'])
+ACCOUNT_REQUESTS = Counter('ve_manager_account_invocations', 'Number of calls per account', ['account'])
```

Comment:
Why do not add another label to REQUST_COUNTS?

Reply:
Adding another label to REQUEST_COUNTS causes that metric to have one other dimension (up to 3 from 2) so that metric is then indexed by three elements.
```
ve_manager_invocations_total{account="012345",endpoint="/v1/business_risk",method="get"} 1.0
ve_manager_invocations_total{account="0",endpoint="/v1/business_risk",method="get"} 2.0
```
Which destroys the metric and bringing it back to the state preceding such change would require some grouping by.

As opposed to single specialized metric for account request counts:
```
ve_manager_account_invocations_total{account="0"} 14.0
ve_manager_account_invocations_total{account="0afdsf"} 1.0
ve_manager_account_invocations_total{account="012345"} 1.0
```

Additionally this way the change was smaller as adding label to REQUEST_COUNTS would require changes at 4 places as opposed to one.